frontend/packages/console-shared/src/components/ReleaseBadge.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { Label } from '@patternfly/react-core';
+
+export const TechPreviewBadge: React.FC = () => (
+  <Label style={{ backgroundColor: '#D93F00' }}>Tech Preview</Label>
+);
+
+export const DevPreviewBadge: React.FC = () => (
+  <Label style={{ backgroundColor: '#D93F00' }}>Dev Preview</Label>
+);
+
+export const ReleaseBadge: React.FC<ReleaseBadgeProps> = ({ releaseBadgeType }) => (
```

Comment:
This component wouldn't be necessary if there was no string to bade mapping.

Reply:
I agree here. This could quickly get messy if we add more badges in future. It's better if parent sends the . component directly instead of sending badge type prop.